engine/src/main/java/org/entando/entando/web/common/model/RestListRequest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.common.model;
+
+import java.util.Arrays;
+
+import com.agiletec.aps.system.common.FieldSearchFilter;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+public class RestListRequest {
+
+    private String sort;
+    private String direction;
+
+    private Integer pageNum = 0;
+    private Integer pageSize = 5;
+
+    private Filter[] filter;
```

Comment:
If possible lets make this plural. `filters` May have to us an annotatation for the json conversion.